prefect_aws/client_wait.py

```diff
@@ -0,0 +1,82 @@
+"""Task for waiting on a long-running AWS job"""
+
+import importlib.resources as pkg_resources
+import json
+from functools import partial
+from typing import Any, Dict, Optional
+
+import boto3
+from anyio import to_thread
+from botocore.exceptions import WaiterError
+from botocore.waiter import WaiterModel, create_waiter_with_client
+from prefect import get_run_logger, task
+
+from prefect_aws import waiters
+from prefect_aws.credentials import AwsCredentials
+
+
+@task
+async def client_waiter(
+    client: str,
+    waiter_name: str,
+    waiter_definition: Dict[str, Any],
```

Comment:
I think this should be optional.
```suggestion
    waiter_definition: Optional[Dict[str, Any]] = None,
```

Reply:
Changed to optional, creates a problem adding the =None

Reply:
can you elaborate what the problem is?

Reply:
Actually, think I figured out a way to make it work.  Will let you know tomorrow, just had to change the order of the arguments.